packages/a11y/src/browser/run.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Registry from 'metadata-tools';
+import axe from 'axe-core';
+
+interface ITest {
+  render: (container: HTMLElement) => void;
+  cleanup: () => void;
+}
+
+function createTestsFromSimulations(reactRoot: any) {
+  const tests: ITest[] = [];
+  for (const [Comp, meta] of Registry.metadata.components.entries()) {
+    meta.simulations.forEach((sim) => {
+      tests.push({
+        render:  (container: HTMLElement) => ReactDOM.render(<div id={Comp.name}><Comp {...sim.props} /></div>, container),
+        cleanup: () => ReactDOM.unmountComponentAtNode(reactRoot)
+      });
+    });
+  }
+  return tests;
+}
+
+const root = document.getElementById('react-root');
+async function test(rootElement: HTMLElement) {
+  const comps = createTestsFromSimulations(root);
+  for (const c of comps) {
+    const div = document.createElement('div');
+    rootElement.appendChild(div);
+    await c.render(div);
+  }
+  axe.run(rootElement, (err: Error, result: axe.AxeResults) => {
```

Comment:
It would we very helpful if the errors contained information about which component and which simulation has lead to the error. One way to do that is to run tests one by one (unless this turns out to be really slow).